### ISSUE TITLE
[Bug] Add new tab plus button is impossible to see in private mode when using light theme and accessibility settings (ex: TalkBack) is on

### ISSUE THREAD

**Issue by abhijitvalluri** [mozilla-mobile/fenix#14950]
## Steps to reproduce
1. Enable light theme in Firefox. 
1. Turn on TalkBack in the system settings on your Android device.
1. Now, in Firefox, enable "private mode" either by clicking the eye mask icon on the top right of the home page, OR by opening the tabs tray and clicking the private tabs icon and clicking the new tab button. 
    * Note: please test with the latest master of Firefox or any build that includes the fix for #14540, otherwise the tabs tray will simply crash the app if you have an open tab!
1. Now, in private tabs mode, open the tabs tray again. Note that the + sign in the top right of the tray is nearly invisible. This is because it uses white color, which is the primary text color in private mode.

### Expected behavior

I expect that the new tab plus button is visible in the tabs tray in all scenarios.

### Actual behavior

For the specific scenario of using a light theme and in private tabs mode, with accessibility settings enabled (ex: TalkBack), the button is impossible to see. See the screenshot below. If you download the image and inspect closely, you can see a white button on a very light background. The button exists and talkback can highlight it in green as seen. Also, note that the issue does not exist in a dark theme or in regular tabs mode.

#### Screenshots
<img src="https://user-images.githubusercontent.com/17151327/92746622-558c3380-f37b-11ea-9e14-fffa2f789e31.png" width="350" /> <img src="https://user-images.githubusercontent.com/17151327/92746647-5ae97e00-f37b-11ea-9413-fd336c5506cd.png" width="350" />

#### Cause for the issue
The culprit is: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/drawable/ic_new.xml#L11
```
        android:fillColor="?primaryText"
```
should instead be:
```
        android:fillColor="@color/primary_text_normal_theme"
```

### Device information

* Android device: Pixel 2, running Android 10
* Fenix version: Issue still exists on the latest master 2e3863578187d7fce4ca8e90b26ace165c1ea15c

**Comment by abhijitvalluri** [comment]
#14951 is my fix for this issue. Please let me know if this is the correct way to create the PR. Let me know if there is a better way to fix this! Thanks!

**Comment by yoasif** [comment]
May be a dupe of https://github.com/mozilla-mobile/fenix/issues/14946.

**Comment by abhijitvalluri** [comment]
Indeed it appears to be a dupe of that one. In fact, I noticed this bug soon after raising my previous bug report about issues with accessibility in the tabs tray component: #14540. I noticed soon after then that this issue also exists and found the fix (I believe it is the right fix). But I waited till that other issue was fixed as it was a bigger issue and this issue is not noticeable as the app crashed before reaching this point. Feel free to consolidate this bug report with the other.
